pages/index.ejs

```diff
@@ -1,11 +1,46 @@
 ---
-title: Homepage
-description: Get your webiste up and running online in no time using current starter
-header: <fix>CMintS</fix> starter
+title: Privacy Manager
+description: Privacy Manager website
 ---
 
-<h2>{homepage[Page heading] Homepage}</h2>
-
-<p>{homepage-p[Paragraph in "Homepage" section] This is the homepage that is using <fix>EJS</fix>.}</p>
-
-<a href="/page2">{homepage-p-link[Link to the second page in "Homepage" section] Go to Page2}</a>
+<section class="col2 container">
+  <div>
+    <h2>{privacy-manager[Header of the section] Privacy Manager}</h2>
+    <ul>
+      <li>{privacy-manager-li1[Privacy Manager section bullet point] Privacy Management}</li>
+      <li>{privacy-manager-li2[Privacy Manager section bullet point] <fix>Cookie</fix> Management}</li>
+      <li>{privacy-manager-li3[Privacy Manager section bullet point] Browsing data deletion on browser start}</li>
+      <li>{privacy-manager-li4[Privacy Manager section bullet point] Network monitoring}</li>
+    </ul>
+    <a id="install-button" href="https://chrome.google.com/webstore/detail/privacy-manager/giccehglhacakcfemddmfhdkahamfcmd" target=_blank>Get Privacy Manager for Chrome</a>
```

Comment:
Detail: Use translation string for `Get Privacy Manager for Chrome `.

Reply:
Suggestion: `Privacy-manager-link[Link to Chrome extension web store] `

Reply:
`Privacy-manager-download-button[Download button text in 'Privacy Manager' section]`